test/server/reporters-test.js

```diff
@@ -336,4 +352,12 @@ describe('Reporters', function () {
     it('xunit', function (done) {
         testReporter('xunit', done);
     });
+
+    it('Should xunit reporter use plain text decorator', function () {
+        testUsingPlainTextDecorator('xunit');
+    });
+
+    it('Should json reporter use plain text decorator', function () {
```

Comment:
They actually can be combined in one test, test name should include reference to github issue (in format `GH-{num}`) and meaningful name (check it with @VasilyStrelyaev )